CHANGELOG.md

```diff
@@ -1,11 +1,54 @@
+# 4.0.0 (2020-05-11)
+
+### New features
+
+- Updated the pipeline to transpile to the [Angular Package Format](https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/preview). [#51](https://github.com/blackbaud/skyux-i18n/pull/51)
+
+### Breaking changes
+- Dropped support for `rxjs@5`. Consumers can install `rxjs-compat@^6` to support older versions of `rxjs`. [#51](https://github.com/blackbaud/skyux-i18n/pull/51)
+- Removed `SkyAppHostLocaleProvider`. This class is provided automatically by `@skyux-sdk/builder@>=3`. [#51](https://github.com/blackbaud/skyux-i18n/pull/51)
```

Comment:
We should probably add an article after Removed in both this line and the next.

Reply:
We usually don't add articles before code snippets like that when they standalone. If the code snippets are used as adjectives following a noun right after the snippet, then we use articles. But we generally don't when the code snippet stands alone.

Reply:
Disregard my comments!